app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+    @post_id = @post.id
```

Comment:
don't think you need to have a separate instance variable for this...can just reference `@post.id`...